pkgs/tools/security/shellz/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "shellz";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "evilsocket";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1mhl1y0jkycyl1hgwxavxkm1f6kdx1sz3bvpmkr46sdijji06imi";
+  };
+
+  vendorSha256 = "14rd9xd7s5sfmxgv5p9ka8x12xcimv5hrq7hzy0d1c3ddf50rr7n";
```

included in upstream makefile, see https://github.com/evilsocket/shellz/blob/v1.6.0/Makefile#L6
```suggestion
  vendorSha256 = "14rd9xd7s5sfmxgv5p9ka8x12xcimv5hrq7hzy0d1c3ddf50rr7n";
  
  ldflags = [ "-s" "-w" ];
```

---

Would only be a 4 MB win (approx. 9 %) for us if we strip the debugging information. As far as I remember is the DWARF symbol table omitted by default in Nix or at least the output looks like that nowadays.

```bash
...:~/shellz]$ du -h shellz-*
35M	shellz-s-w
39M	shellz-w
48M	shellz-unstripped
```